x-pack/plugins/ml/public/application/overview/components/analytics_panel/analytics_panel.tsx

```diff
@@ -93,48 +89,82 @@ export const AnalyticsPanel: FC<Props> = ({
 
   const panelClass = isInitialized === false ? 'mlOverviewPanel__isLoading' : 'mlOverviewPanel';
 
+  const transformsLink = `${basePath.get()}/app/management/data/transform`;
+
+  const noDFAJobs = errorMessage === undefined && isInitialized === true && analytics.length === 0;
+
+  if (noDFAJobs) {
+    return (
+      <EuiEmptyPrompt
+        layout="horizontal"
+        hasBorder={true}
+        hasShadow={false}
+        icon={<EuiImage size="fullWidth" src={adImage} alt="anomaly_detection" />}
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.ml.overview.analyticsList.createFirstJobMessage"
+              defaultMessage="Create your first data frame analytics job"
+            />
+          </h2>
+        }
+        body={
+          <>
+            <p>
+              <FormattedMessage
+                id="xpack.ml.overview.analyticsList.emptyPromptText"
+                defaultMessage="Data frame analytics enables you to perform outlier detection, regression, or classification analysis on your data and annotates it with the results. The job puts the annotated data and a copy of the source data in a new index."
+              />
+            </p>
+            <EuiCallOut
+              size="s"
+              title={
+                <FormattedMessage
+                  id="xpack.ml.overview.analyticsList.emptyPromptHelperText"
+                  defaultMessage="We recommend using {transforms} to create feature indices for analytics jobs."
```

Comment:
> We recommend using {transforms} to create feature indices for analytics jobs.

In the screenshots this is appearing out of context of DFA and so seems like an odd placement IMO. It also seems like a pretty low-level detail for an overview page, unless the goal is advertising transforms.  If it's kept, I'd suggest wording along these lines to avoid the "we":

Data frame analytics requires specifically structured source data; use transforms to create data frames before you create the jobs.

Reply:
Two important moments to point out: 

1. I used the original text we had in the Overview page sidebar
2. On the new Overview page it's currently mentioned in 2 places. I agree it's odd having in the general intro, remove it from there.
I also put it into the empty state of the DFA section. 
![image](https://user-images.githubusercontent.com/5236598/146173495-5792123c-3e45-470a-a656-22eaed665721.png)

So based on your suggestions:
- Info callout 
![image](https://user-images.githubusercontent.com/5236598/146174280-9993681b-a2cd-4856-bb0c-2c9ae8f6664d.png)

- DFA empty state
![image](https://user-images.githubusercontent.com/5236598/146174728-52327e0d-92b9-4f0b-969b-2bf0131305a7.png)